Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -53,9 +86,9 @@ public void TearDown()
         #region Tests
 
         [UnityTest]
-        public IEnumerator ButtonInstantiate()
+        public IEnumerator ButtonInstantiate([ValueSource(nameof(PressableButtonsTestPrefabFilenames))] string prefabFilename)
```

Comment:
Woah cool

Reply:
I know.  It's kind of like magic.  I learned about this relatively recently when working on [UnityGLTF](https://github.com/KhronosGroup/UnityGLTF).  Unity knows how to parse this in its test window and it shows up like this.

![image](https://user-images.githubusercontent.com/41760870/64276738-7d8dbf00-cefd-11e9-97fb-8a8915bc845f.png)